set.mm

```diff
@@ -147154,6 +147148,405 @@ of an open range of nonnegative integers (of length equal to the length of
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Prefixes of a word
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  A prefix is an arbitrary substring/subword starting at the
+  beginning of a string/word (see Wikipedia "Substring"
+  ~ https://en.wikipedia.org/wiki/Substring#Prefix or
+  ~ https://math.stackexchange.com/questions/2190559/
+  is-there-standard-terminology-notation-for-the-prefix-of-a-word ).
+
+  This is the meaning of "prefix" in computer science (automata theory or
+  formal languages) , whereas in linguistics, a prefix has a meaning (see
+  Wikipedia "Prefix" ~ https://en.wikipedia.org/wiki/Prefix ).
+
+$)
+
+  $c prefix $.
+
+  $( Syntax for the prefix operator. $)
+  cpfx $a class prefix $.
+
+  ${
+    $d l s $.
+    $( Define an operation which extracts prefixes of words, i.e. subwords
+       starting at the beginning of a word.  Definition in Section 9.1 of
+       [AhoHopUll] p. 318.  "pfx" is used as label fragment (as propose in
+       ~ https://www.allacronyms.com/prefix/abbreviated .  (Contributed by AV,
```

Comment:
"proposeD"
But I think you can remove that parenthesis anyway (there is no closing parenthesis, by the way). There is no need to justify our label choices by websites, especially nonmathematical ones.

Reply:
> "proposeD" But I think you can remove that parenthesis anyway (there is no closing parenthesis, by the way). 

Done

>There is no need to justify our label choices by websites, especially nonmathematical ones.

I added this because there was a long discussion about the label in #1648.